connectors/src/connectors/intercom/lib/conversation_permissions.ts

```diff
@@ -141,11 +141,18 @@ export async function retrieveIntercomConversationsPermissions({
       });
     }
   } else {
+    const teams = await fetchIntercomTeams(intercomClient);
     if (isRootLevel) {
+      // This is an ugly hack
+      // Since we only sync conversations attached to a Team, if there are no teams, we display "No conversations"
+      if (teams.length === 0) {
+        rootConversationNode.title =
+          "No conversations available for sync (Dust only supports conversations attached to a Team)";
```

Comment:
Let's remove the parenthesis and add that to the connector limitations if not already there